style.css

```diff
@@ -124,81 +190,174 @@ hr {
     aspect-ratio: 1;
     border-radius: 1em;
     background-color: var(--nord-snow-storm-2);
+
+    img {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        left: 0;
+        margin: auto;
+        object-fit: scale-down;
+        object-position: center;
+        width: calc(100% - (var(--icon-padding) * 2));
+        height: calc(100% - (var(--icon-padding) * 2));
+    }
 }
-.about-li-icon img {
-    position: absolute;
-    top: 0;
-    bottom: 0;
-    right: 0;
-    left: 0;
-    margin: auto;
-    object-fit: scale-down;
-    object-position: center;
-    width: calc(100% - (var(--icon-padding) * 2));
-    height: calc(100% - (var(--icon-padding) * 2));
-}
-#experience ul {
-    padding-left: 0;
+
+#experience {
+    ul {
+        padding-left: 0;
+    }
 }
-#education {
-    margin-bottom: 4em;
+
+#education,
+#experience {
+    margin-bottom: var(--spacer);
 }
-#education ul {
-    padding-left: 0;
+
+#education {
+    ul {
+        padding-left: 0;
+    }
 }
 
 /* Projects page */
-.projects-li {
-    margin-top: 2em;
+#projects-content {
+    h2 {
+        margin-top: calc(var(--spacer) / 2);
+    }
 }
-.projects-project-title {
-    vertical-align: bottom;
-    display: inline;
+
+.projects-li {
+    margin-top: calc(var(--spacer) / 2);
 }
+
 .project-header {
     display: flex;
     align-items: flex-end;
+    gap: 1em;
+    justify-content: space-between;
     border-bottom: 1px solid var(--color-fg);
-    margin-bottom: 2em;
+    margin-bottom: calc(var(--spacer) / 4);
     padding-bottom: 0.2em;
+
+    >a {
+        text-decoration: none;
+        flex-basis: 33%;
+
+        &:visited {
+            color: var(--color-link);
+        }
+
+        &:hover,
+        &:focus-visible {
+            color: var(--color-link-hover);
+        }
+
+        h3 {
+            font-size: var(--font-size-med);
+            margin: 0;
+
+            &::before {
```

Comment:
Take this or leave it, but I put a lil `>` before each of the h3's so the list looks like a terminal :)